.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Changes
+
+<!--
+  Required.
+
+  Briefly describe the changes introduced by this pull request.
+  Also link relevant issues with the "closes", "fixes" or "resolves" keywords,
+  and add co-authors where appropriate with the "co-authored-by" keyword.
+
+  Example:
+  Implemented the functionality to update the user's name.
+  Closes #42
+  Co-authored-by: @octocat
+-->
+
+## Implementation details
+
+<!--
+  Optional.
+
+  If useful for the code review, describe implementation details and, if
+  necessary, why a certain approach was chosen.
+
+  Example:
+  Changes are introduced to the controller and the database model. UI changes
+  are not in the scope of this PR.
+-->
+
+## How to read this PR
+
+<!--
+  Optional.
+
+  If useful for the code review, add some guidance for how to read this PR,
+  including links to preview builds.
```

Comment:
Maybe we can merge this section with the `Implementation details` section? What do you think? To have less sections.

Reply:
Maaaaybe 🤔... I like the part of having less sections, but I also think that they are different enough to have their own to be honest.